R/board_rsconnect.R

```diff
@@ -587,7 +592,18 @@ update_cache <- function(path, key, value) {
 
 rsc_path <- function(board, path) {
   board_path <- httr::parse_url(board$url)$path
-  paste0("/", board_path, "/", path)
+  file.path(board_path, path, fsep = "/")
+}
+
+
+rsc_ensure_path_url <- function(board, url) {
+  if (length(url) == 0) return(url)
+  server_name <- dirname(board$server_name)
+  full_name <- board$server_name
+  if (!grepl(full_name, url)) {
```

Comment:
Should probably use `fixed = TRUE` here to be safe

Reply:
Fixed.